CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## SL21d_1 - 2021-12-01
+
+### Enhancements
+
+- FCS
+  - Fcs mudst ([#200](https://github.com/star-bnl/star-sw/pull/200))
+- sTGC
+  - StEvent additions for sTGC (Ftt) detector ([#209](https://github.com/star-bnl/star-sw/pull/209))
+- Online
+  - Trigger group removed 5DSM from BBC crate, thus changing structure. ([#207](https://github.com/star-bnl/star-sw/pull/207))
+  - StEvent StTriggerData2022 update associate with  ([#203](https://github.com/star-bnl/star-sw/pull/203))
+  - updates to daq readers in prep for run 22 ([#186](https://github.com/star-bnl/star-sw/pull/186))
+
+### Bug fixes
+
+- EPD
+  - Fix cut limits in EPD macro ([#188](https://github.com/star-bnl/star-sw/pull/188))
+- FCS
+  - StFcsDB makes no file unless debug is on ([#218](https://github.com/star-bnl/star-sw/pull/218))
+- Online
+  - Fix for pulser histogram in online plots ([#187](https://github.com/star-bnl/star-sw/pull/187))
+- TPC
+  - Move back to MagFactor instead of starMagOnl for pre 2012 data (issue #185) (#216)
```

Comment:
```suggestion
  - Move back to MagFactor instead of starMagOnl for pre 2012 data (issue [#185](https://github.com/star-bnl/star-sw/issues/185)) ([#216](https://github.com/star-bnl/star-sw/pull/216))
```